hw/audio/ac97.c

```diff
@@ -434,6 +462,12 @@ static void update_combined_volume_out (AC97LinkState *s)
     rvol = (rvol * prvol) / 255;
 
     AUD_set_volume_out (s->voice_po, mute, lvol, rvol);
+
+    mute = 1;
+    lvol = 0;
+    rvol = 0;
+
+    AUD_set_volume_out (s->voice_so, mute, lvol, rvol);
```

Comment:
This forces silence on "ac97.so" (digital audio port).

Ideally we'd have some S/PDIF decoder in XQEMU-Manager and find a way to choose between "ac97.po" (analog audio port) and this one.

I'll create an issue on xqemu/xqemu or xqemu/xqemu-manager after merge.